server/src/services/event/EventService.js

```diff
@@ -0,0 +1,50 @@
+const dayjs = require('dayjs');
+const timeSpans = require('./timeSpans');
+const scopes = require('./scopes');
+
+const stores = [
+    function (prefix, redisService, key, userId) {
+        return redisService.setBit(`${prefix}:bitmap:${key}`, userId, 1);
+    },
+
+    async function (prefix, redisService, key) {
+        return redisService.increment(`${prefix}:increment:${key}`);
+    },
+
+    async function (prefix, redisService, key, userId) {
+        return redisService.addToSet(`${prefix}:set:${key}`, userId);
+    }
+];
+
+class EventService {
+    constructor(prefix, redisService) {
+        this.prefix = prefix;
+        this.redisService = redisService;
+    }
+
```

Comment:
I would add here:
get stores() {
    return [
        ... here add functions ...
    ];
}

Reply:
and then you don't need to pass this.redisService, because it will be already there